src/containers/Resources/Resources.tsx

```diff
@@ -40,6 +40,9 @@ const StyledNav = styled("nav", {
     display: "flex",
     flexDirection: "column",
     gap: "medium",
+    "& > :first-child": {
+      marginBlockEnd: "-xsmall",
+    },
```

Comment:
Hvorfor trenger vi dette? Generelt sett lite fan av negative marginer

Reply:
For første child skal ha halvparten av gap sammenlignet med resten. Tenkte kanskje dette var litt enklere enn å omstrukturere

Reply:
Enklere nå, men ikke nødvendigvis så fremtidssikkert? Kanskje vi heller skal omstrukturere?